assets/scripts/index.js

```diff
@@ -43,9 +64,36 @@ function division(num1,num2){
     num2 = parseInt(document.getElementById('number2').value);
     result = num1 / num2;
     document.getElementById('result').innerHTML = result;
+    button='save';
+    show();
+}
+
+function save_result(){
+    saved_results.push(result)
+    button = 'show';
+    show();
+}
+
+function show_results(){
+    let text = '<ul>';
+    for (x=0;x<saved_results.length;x++){
```

Comment:
Este for puede ser cambiado por un `map` para utilizar sintaxis de ES6 , que es mas rapido y mejor de leer.

https://www.w3schools.com/jsref/jsref_foreach.asp